src/Reflection/ReflectionParameter.php

```diff
@@ -99,6 +99,19 @@ public static function createFromClassInstanceAndMethod($instance, $methodName,
     }
 
     /**
+     * Create a reflection of a parameter using a closure
+     *
+     * @param \Closure $closure
+     * @param string $parameterName
+     * @return ReflectionParameter
+     */
+    public static function createFormClosure(\Closure $closure, $parameterName)
```

Comment:
* typo in method name `createFromClosure`
 * also please add type declarations (`string $parameterName` and ` : ReflectionParameter`) (I'm working on adding type declarations everywhere else as we're min PHP 7 for next release now)